web/app/js/components/Deployment.jsx

```diff
@@ -110,9 +115,14 @@ export default class Deployment extends React.Component {
           pendingRequests: false,
           loaded: true
         });
```

Comment:
Should this call to `setState` also set `error: ''`?

Reply:
ah yes, good catch!